### PR TITLE
Add documentation for `--minimum-expected-tests` `--filter` and `--settings`

### LINKED ISSUE #2684: Add documentation for `--minimum-expected-tests` `--filter` and `--settings`
Add documentation for `--minimum-expected-tests` here https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-intro?tabs=dotnetcli#options

 `--filter` and `--settings` here https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-runner-intro

### PR DESCRIPTION
**Pull request by MarcoRossignoli**
Closes https://github.com/microsoft/testfx/issues/2684

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/testing/unit-testing-mstest-runner-intro.md](https://github.com/dotnet/docs/blob/466402d5af3f588edbf3b00b34d556da5af6b209/docs/core/testing/unit-testing-mstest-runner-intro.md) | [MSTest runner overview](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-runner-intro?branch=pr-en-us-40377) |
| [docs/core/testing/unit-testing-platform-intro.md](https://github.com/dotnet/docs/blob/466402d5af3f588edbf3b00b34d556da5af6b209/docs/core/testing/unit-testing-platform-intro.md) | [Microsoft.Testing.Platform overview](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-platform-intro?branch=pr-en-us-40377) |

### CODE REVIEW

**Comment by MarcoRossignoli** [comment]
cc: @Evangelink @nohwnd

**Review by Evangelink — APPROVED** [review]

**Comment by MarcoRossignoli** [comment]
@IEvangelist this can be merged

File: docs/core/testing/unit-testing-mstest-runner-intro.md [hunk 1]
```diff
@@ -55,3 +55,61 @@ To enable the MSTest runner in a MSTest project, you need to add the `EnableMSTe
...
+
+using the executable
+
+```bash
+Contoso.MyTests.exe --settings config.runsettings
+```
+
+### Tests filter
+
+Seamesly you can provite the tests [filter](https://learn.microsoft.com/dotnet/core/testing/selective-unit-tests?pivots=mstest#mstest-examples) using the command line option `--filter`, for instance:
```
Comment (Evangelink) [inline docs/core/testing/unit-testing-mstest-runner-intro.md:91]:
```suggestion
Seamesly you can provite the tests [filter](./selective-unit-tests#mstest-examples) using the command line option `--filter`, for instance:
```

File: docs/core/testing/unit-testing-mstest-runner-intro.md [hunk 2]
```diff
@@ -55,3 +55,61 @@ To enable the MSTest runner in a MSTest project, you need to add the `EnableMSTe
...
+
+## Configurations and filters
+
+### .runsettings
+
+The MSTest runner supports the [runsettings](unit-testing-platform-runsettings.md) through command line option `--settings`, for instance:
+
+using `dotnet run`
+
+```bash
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-runner-intro.md:67]:
```suggestion
```dotnetcli
```

File: docs/core/testing/unit-testing-mstest-runner-intro.md [hunk 3]
```diff
@@ -55,3 +55,61 @@ To enable the MSTest runner in a MSTest project, you need to add the `EnableMSTe
...
+dotnet run --project Contoso.MyTests -- --settings config.runsettings
+```
+
+using `dotnet exec`
+
+```bash
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-runner-intro.md:73]:
```suggestion
```dotnetcli
```

File: docs/core/testing/unit-testing-mstest-runner-intro.md [hunk 4]
```diff
@@ -55,3 +55,61 @@ To enable the MSTest runner in a MSTest project, you need to add the `EnableMSTe
...
+dotnet exec Contoso.MyTests.dll --settings config.runsettings
+```
+
+or
+
+```bash
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-runner-intro.md:79]:
```suggestion
```dotnetcli
```

File: docs/core/testing/unit-testing-mstest-runner-intro.md [hunk 5]
```diff
@@ -55,3 +55,61 @@ To enable the MSTest runner in a MSTest project, you need to add the `EnableMSTe
...
+dotnet Contoso.MyTests.dll --settings config.runsettings
+```
+
+using the executable
+
+```bash
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-runner-intro.md:85]:
```suggestion
```dotnetcli
```

File: docs/core/testing/unit-testing-mstest-runner-intro.md [hunk 6]
```diff
@@ -55,3 +55,61 @@ To enable the MSTest runner in a MSTest project, you need to add the `EnableMSTe
...
+Contoso.MyTests.exe --settings config.runsettings
+```
+
+### Tests filter
+
+Seamesly you can provite the tests [filter](./selective-unit-tests#mstest-examples) using the command line option `--filter`, for instance:
+
+using `dotnet run`
+
+```bash
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-runner-intro.md:95]:
```suggestion
```dotnetcli
```

File: docs/core/testing/unit-testing-mstest-runner-intro.md [hunk 7]
```diff
@@ -55,3 +55,61 @@ To enable the MSTest runner in a MSTest project, you need to add the `EnableMSTe
...
+
+or
+
+```bash
+dotnet Contoso.MyTests.dll --filter "FullyQualifiedName~UnitTest1|TestCategory=CategoryA"
+```
+
+using the executable
+
+```bash
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-runner-intro.md:113]:
```suggestion
```dotnetcli
```

**Review by IEvangelist — APPROVED** [review]

File: docs/core/testing/unit-testing-mstest-runner-intro.md [hunk 8]
```diff
@@ -55,3 +55,61 @@ To enable the MSTest runner in a MSTest project, you need to add the `EnableMSTe
...
+Contoso.MyTests.exe --settings config.runsettings
+```
+
+### Tests filter
+
+Seamesly you can provite the tests [filter](./selective-unit-tests#mstest-examples) using the command line option `--filter`, for instance:
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-runner-intro.md:91]: This link is incorrect, please fix this - not sure where it should point to?

Comment (MarcoRossignoli) [reply docs/core/testing/unit-testing-mstest-runner-intro.md:91]: This one https://learn.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests?pivots=mstest#mstest-examples looks in the same folder or?

**Review by IEvangelist — APPROVED** [review]
This fixes the issue.

**Pull request merged**